spec/ttml1.xml

```diff
@@ -4598,22 +4598,22 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
```

Comment:
This additional sentence is unclear and confusing in its introduction of the term 'relative' out of the blue as it were. We need to reconsider what we are trying to do/say here then find a better formulation for doing so.

Reply:
@skynavga The term "relative" is key since 8.4.4.3(4) specifies that only relative value types, e.g. "%" or "em", have their specified value replaced by their computed values in the _ Computed Style Set_, which is inherited in 8.4.4.2(c). In this case, we do not want "normal" to be replaced by its computed value, so it is sufficient to clarify that it is not a relative value.